understanding/accessible-authentication-enhanced.html

```diff
@@ -53,136 +53,115 @@
                </div>
             </nav>
          </aside>
          <main id="main" class="standalone-resource__main">
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				SC 3.3.9:</span>Accessible Authentication (Enhanced) (Level AAA)
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				達成基準 3.3.9:</span>アクセシブルな認証 (高度) (レベル AAA)
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
-                  <p>A <a href="#dfn-cognitive-function-test">cognitive function test</a> (such as remembering a password or solving a puzzle) is not required for any step
-                     in an authentication <a href="#dfn-process">process</a> unless that step provides at least one of the following:
+                  <p><a href="#dfn-cognitive-function-test">認知機能テスト</a> (パスワードを記憶する、パズルを解く、など) は、認証<a href="#dfn-process">プロセス</a>のどのステップにおいても要求されない。ただし、そのステップが次の少なくとも一つを提供する場合は除く。
                   </p>
                   <dl>
                      
-                     <dt>Alternative</dt>
+                     <dt>代替手段</dt>
                      
-                     <dd>Another authentication method that does not rely on a cognitive function test.</dd>
+                     <dd>認知機能テストに依存しない別の認証方法がある。</dd>
                      
-                     <dt>Mechanism</dt>
+                     <dt>メカニズム</dt>
                      
-                     <dd>A <a href="#dfn-mechanism">mechanism</a> is available to assist the user in completing the cognitive function test.
+                     <dd>認知機能テストを利用者が完遂できるように支援する<a href="#dfn-mechanism">メカニズム</a>が利用できる。
                      </dd>
                      
                   </dl>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>意図</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Make logins possible with less mental effort.</dd>
+                  <dt>目標</dt>
+                  <dd>より少ない精神的労力でログインできるようにする。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Don't make people recognize objects or user-supplied images and media to login.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>ログインするために、人々に物体又は利用者が提供した画像及びメディアを認識させない。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>Some people with cognitive disabilities can't do puzzles, including identifying objects
-                     and non-text information they previously supplied.
+                  <dt>なぜそれが重要か</dt>
+                  <dd>認知障害のある人々の中には、以前に提供した物体及び非テキストの情報を特定するなどの、パズルができない人もいる。
```

Comment:
達成基準 3.3.8 の例外事項を見ると、「物体の認識」と「個人特有のコンテンツ（i.e. その利用者本人がウェブサイトに提供した非テキストコンテンツ）」という建付けになっているので、原文の they previously supplied は non-text information のみにかかっていて、objects にはかかっていないように思います。以下はいかがでしょうか？

 ```suggestion
                  <dd>認知障害のある人々の中には、物体及び以前に自身が提供した非テキストの情報を特定するなどの、パズルができない人もいる。
```